.storybook/Input.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Box, Input, Label, theme } from '../src'
+
+storiesOf('Input', module)
+  .add('Input component', withInfo({
+    inline: true,
+    text: 'Simple styled input component that accepts a color and whether or not to show an error container.'
+  })(() => (
+    <Label m={3}>
+      Label Component
+    </Label>
+  )))
+  .add('Colors', () => (
+    <Box width={400}>
+      <Input mb={3} id="input-colors-1" placeholder="No color"/>
+      <Input mb={3} id="input-colors-2" color={theme.colors.red} placeholder="Red"/>
+      <Input mb={3} id="input-colors-3" color={theme.colors.green} placeholder="Green"/>
```

Comment:
Haven't looked too closely at the implementation yet, but the styled-system `color` function picks up colors from `props.theme` so that you can set the key as a prop value, e.g. `color='green'` would give you `theme.colors.green`. I think we'd want this `color` prop to work the same way